tests/python-gpu/test_large_input.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+import xgboost as xgb
+import cupy as cp
+import time
+import pytest
+
+
+# Test for integer overflow or out of memory exceptions
+def test_large_input():
+    available_bytes, _ = cp.cuda.runtime.memGetInfo()

```

Comment:
Currently, this test will skip when `--use-rmm-pool` is set, as cuPy is not configured with RMM. To make it work with RMM, we'll need to run
```python
cupy.cuda.set_allocator(rmm.rmm_cupy_allocator)
```

Do we want to enable this test with RMM?

Reply:
Don't think this is necessary for now.